### PR TITLE
store/tikv: Make maxTxnTimeUse a configurable value 

### PR DESCRIPTION
**Pull request by qw4990**
cherry-pick for #8215

### CODE REVIEW

**Comment by jackysp** [comment]
I think if there is no customer requires to use it in Release 2.1, we should not cherry-pick it. There are some potential risks if it is not set properly.

**Comment by MyonKeminta** [comment]
@zz-jason PTAL

**Comment by zz-jason** [comment]
LGTM

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -159,20 +166,23 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 		return nil, errors.Trace(err)
 	}
 
+	// Convert from sec to ms
+	maxTxnTimeUse := uint64(config.GetGlobalConfig().TiKVClient.MaxTxnTimeUse) * 1000
 	commitDetail := &execdetails.CommitDetails{WriteSize: size, WriteKeys: len(keys)}
-	metrics.TiKVTxnWriteKVCountHistogram.Observe(float64(commitDetail.WriteKeys))
-	metrics.TiKVTxnWriteSizeHistogram.Observe(float64(commitDetail.WriteSize))
+	metrics.TiKVTxnWriteKVCountHistogram.Observe(float64(len(keys)))
+	metrics.TiKVTxnWriteSizeHistogram.Observe(float64(size))
```
Comment (MyonKeminta) [inline store/tikv/2pc.go]: Is this change caused by incorrectly resolving git's conflict? This change is not part of the previous PR.

Comment (zz-jason) [reply store/tikv/2pc.go]: @qw4990 PTAL

Comment (qw4990) [reply store/tikv/2pc.go]: My mistake.
Some invalid changes have been removed now, PTAL @zz-jason @MyonKeminta

**Comment by qw4990** [comment]
/run-all-tests tidb-test=release-2.1 tikv=release-2.1 pd=release-2.1

**Comment by MyonKeminta** [comment]
LGTM

**Comment by MyonKeminta** [comment]
/run-common-test tidb-test=release-2.1 tikv=release-2.1 pd=release-2.1
/run-integration-common-test tidb-test=release-2.1 tikv=release-2.1 pd=release-2.1

**Comment by qw4990** [comment]
/run-all-tests tidb-test=release-2.1 tikv=release-2.1 pd=release-2.1

**Comment by qw4990** [comment]
CI problems have been resolved.
Please approve this PR again @MyonKeminta .

**Comment by qw4990** [comment]
@winkyao This PR is ready to be merged.

**Pull request merged**